cache/ttl/cache.go

```diff
@@ -74,7 +94,11 @@ func (cache *Cache) Get(key string) (interface{}, bool) {
 		cache.mutex.Lock()
 		defer cache.mutex.Unlock()
 		item, exists := cache.items[key]
-		if !exists || item.expired() {
+		if !exists {
+			return nil, false
+		}
+		if item.expired() {
+			delete(cache.items, key)
```

Comment:
Could we keep old code and let the goroutine do all cleanup jobs?

Reply:
we could, but it is not accurate, consider this scenario:
```
time 0: cache = NewCache(time.Second)
time 0.5s: cache.Set("y", 1)
time 1s: goroutine clean-up, "y" is not expired yet
time 1.6s: data, ok := cache.Get("y")
data = 1, ok = true
```
at time 1.6s, "y" actually already expired, but goroutine won't remove it until 2s
so that's why need to also delete here

Reply:
the problem is if we don't delete it here, `Count()` may > 0 even though all items expired, `Count()` won't become 0 until next clean-up is triggered